src/components/Slider/Slider.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+  import { onMount } from 'svelte';
+  import { spring } from 'svelte/motion';
+  import { asSlider } from '../../actions/slider';
+  import {
+    createSlidesXPoints,
+    getIncreasedSlideIndex,
+    getDecreasedSlideIndex
+  } from './helpers';
+  import { setWidthToElement } from '../../helpers/common';
+
+  const x = spring(0, {
+    stiffness: 0.07,
+    damping: 0.35
+  });
+
+  let slider;
+  let sliderContent;
+  let slidesCount;
+  let xOnSlideStart;
+  let slidesXPoints;
+  let activeSlideIndex = 0;
+  // to calculate x on slidemove
+  let currentTranslateX = 0;
+
+  onMount(() => {
```

Comment:
Not sure about svelte, but isn't it should be removeEventListener somewhere in unmount?
So far it looks like a memory leak

Reply:
Shame on me, sure they should be there!
My English teacher told me that if I hurry I often do simple mistakes